### ISSUE TITLE
voila template

### ISSUE THREAD

**Issue by kasrakazemi** [voila-dashboards/voila-vuetify#61]
is there any way or template to make voila responsive?
i just need that my app be responsive and nothing more

**Comment by maartenbreddels** [comment]
Hi,

this one is https://github.com/maartenbreddels/voila-demo
It's not about the template, it's how you add the content.

Regards,

Maarten

**Comment by kasrakazemi** [comment]
you say that it is about placing items and not template?

**Comment by maartenbreddels** [comment]
Yes, using the right vuetify components makes it responsive, not the
template

(from mobile phone)

Op do 28 apr. 2022 22:32 schreef kasrakazemi ***@***.***>:
